scripts/core/superdesk-api.d.ts

```diff
@@ -1025,6 +1025,7 @@ declare module 'superdesk-api' {
         noPadding?: boolean;
         justifyContent?: string;
         bold?: boolean;
+        title?: string;
```

Comment:
This is a layout interface, it'd be good not to include unrelated properties if possible. Can you use the title in the child component?

Reply:
I thought the same but if we use the title in the child property, In this case, we have to declare it in the `<Slugline>` and `<Headline>` Component, So does it make sense to you? Another problem is, that when we hover over the article it will not show the tooltip, it will show the tooltip only if we hover over the slugline or headline which I think is wrong.

Reply:
I took a look at the interface again, and I think adding a title is reasonable because ellipsis is set in the same interface.